l2r_part1_data_eng.ipynb

```diff
@@ -260,15 +328,332 @@
     "- **grade** is the ground truth relevance score for this document, where a grade of '10' is most relevant, while a grade of '1' is least relevant. It is an [ordinal variable](https://en.wikipedia.org/wiki/Ordinal_data) ranging from **1 ~ 10**, from least relevant to most relevant.\n",
     "- **title_en_us** is the English-language title of an article on Microsoft Docs.\n",
     "- **query** is the search query that led to this document\n",
-    "- All features that begin with an \"@\" character are returned from the Features API, and will be used extensively by the trained model.\n",
+    "- All features that begin with an \"@\" character are computed by the Features API and will be used extensively by the trained model. Other features including properties stored in documents including normalized_pageview, tableCount, and sectionCount.\n",
```

Comment:
Grammatical suggestion:

Other features (including -> include) properties stored in documents, (including -> such as) normalized pageview, etc.